### ISSUE TITLE
meta component

### ISSUE THREAD

**Issue by xrado** [mobxjs/mobx-vue#10]
I have a case where I pass component definition directly to vue meta component.

`<component :is="{..component definition}" v-bind="props"/>`

wrapping it with observer breaks a whole thing.
Do you have any idea what to do here?

**Comment by xrado** [comment]
@kuitos I found the issue
if the component has data property with the value null then it breaks on
```
vue.esm.js?a3f3:1741 TypeError: Cannot read property '$mobx' of null
    at eval (collectData.js?eaa5:15)
```
see example https://codesandbox.io/s/xl42xljxrp
click on button "2"

**Comment by kuitos** [comment]
@xrado 
`observer` always returned the extended component constructor, for issue 1 could you provide a codesandbox link?

For issue 2 I will fix it and release soon.

Thanks for your feedback!

**Comment by kuitos** [comment]
@xrado issue 2 fixed and released v2.0.5!

**Comment by xrado** [comment]
@kuitos this also solves the first issue ...awesome, thanks
